### ISSUE TITLE
Incompatibility with LXD 2.13

### ISSUE THREAD

**Issue by dlr2** [conjure-up/conjure-up#866]
## Report

When I try to run the OpenSstack/Nova-LXD spell, I get the _'There is no LXD Bridge found on your system which usually means this is your first time running LXD'_ error.

This was discussed at the LXD forum here: https://github.com/lxc/lxd/issues/3290 which concludes that there should not be request for lxd-bridge for newer LXD installs.

## Please provide the output of the following commands
```
dave@opcoast-ryzen ~ $ which juju
/usr/bin/juju
dave@opcoast-ryzen ~ $ which conjure-up
/usr/bin/conjure-up
dave@opcoast-ryzen ~ $ which lxd
/usr/bin/lxd
dave@opcoast-ryzen ~ $ juju version
2.0.2-genericlinux-amd64
dave@opcoast-ryzen ~ $ conjure-up --version
conjure-up 0.1.2
dave@opcoast-ryzen ~ $ lxd --version
2.13
dave@opcoast-ryzen ~ $ lxc config show
config:
  core.https_address: '[::]:8443'
dave@opcoast-ryzen ~ $ cat /etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18.1
DISTRIB_CODENAME=serena
DISTRIB_DESCRIPTION="Linux Mint 18.1 Serena"
```

## What Spell was Selected?

openstack

## What provider (aws, maas, localhost, etc)?

localhost

## Commands ran

conjure-up openstack

## Additional Information

I didn't include the tarball, since it doesn't seem relevant.

**Comment by battlemidget** [comment]
We ask for the tarball because it is relevant. Could you please attach that?

**Comment by battlemidget** [comment]
A sosreport as well please

**Comment by dlr2** [comment]
OK, based on the prior discussion I downgraded LXD to 2.0.9 and then retried. conjure-up then said that this version was too old and to upgrade. So I then upgraded _back_ to version 2.13 and retried. And it worked.

I have no idea why it didn't work the first time with version 2.13, but its working now, I'm successfully bringing up openstack on localhost as we speak. Thanks for the quick response! And I'll close this one.
